client/src/lib/fetch.js

```diff
@@ -0,0 +1,18 @@
+import { SERVER_URL } from "../constants";
+
+export const getUserData = async (id = "") => {
+  const getUserDataEndpoint = `${SERVER_URL}/user/${id}`;
+  const result = await fetch(getUserDataEndpoint, {
+    credentials: "include",
```

Comment:
Nothing needs to be changed for this PR. But for future, when the server and frontend have the same URL host, we would no longer need this line.